extensions/typescript-language-features/src/features/completions.ts

```diff
@@ -577,16 +577,20 @@ class TypeScriptCompletionItemProvider implements vscode.CompletionItemProvider<
 		};
 
 		const response = await this.client.interruptGetErr(() => this.client.execute('completionEntryDetails', args, token));
+
 		if (response.type !== 'response' || !response.body || !response.body.length) {
 			return item;
 		}
 
+		const definitionResponse = await this.client.interruptGetErr(() => this.client.execute('definition', typeConverters.Position.toFileLocationRequestArgs(filepath, item.position), token));
```

Comment:
This adds an extra request every time we need to resolve a completion. I don't think the benefit this information is worth the cost. Instead, any information required should come from TS as part of the `completionEntryDetails ` response

Reply:
For completion details it probably isn't worth it too often.  99% of the time this will be useful is during the hover, which also has to do this.  It could definitely be removed here and left there to retain almost all the benefits of the PR with relative linking.

Reply:
This would also add an extra request for hovers, which I am also not a fan of since this is also a user facing operation and delays are noticeable. Again, I think any information should come from TS as part of the response